Resources/Prototypes/Entities/Mobs/Species/slime.yml

```diff
@@ -79,6 +79,16 @@
     - BleedersBite
   - type: ReagentDamageModifier
     modifierSet: SlimeReagents
+  - type: Temperature
+    heatDamageThreshold: 422
+    coldDamageThreshold: 285
```

Comment:
This is near-ish to room temperature, are you sure they should start dying at this point? 

I would recommend that you set the damages to be more extreme rather than the temperature limits, and then instead play with the ThermalRegulator component. You can use that to edit their *comfortable* range, which will be a more nuanced effect than just a death-limit.